### PR TITLE
3538: Add validation tests

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/adduser/ui.test.js [hunk 1]
```diff
@@ -173,17 +173,69 @@ test('should navigate to confirmation page when form is filled out and addUser b
...
+  await t
+    .expect(getLocation()).eql('http://localhost:1234/organisations/org1/adduser')
+    .expect(addUserPage.find('[data-test-id="error-summary"]').count).eql(1)
+    .expect(addUserPage.find('li a').count).eql(5)
+    .expect(await extractInnerText(addUserPage.find('#firstName-error'))).contains('First name is required')
+    .expect(addUserPage.find('#lastName-error').count).eql(1)
+    .expect(await extractInnerText(addUserPage.find('#lastName-error'))).contains('Last name is too long')
+    .expect(addUserPage.find('#phoneNumber-error').count).eql(1)
+    .expect(await extractInnerText(addUserPage.find('#phoneNumber-error'))).contains('Telephone number is required')
+    .expect(addUserPage.find('#emailAddress-error').count).eql(1)
```
Comment (alibombaye) [inline app/pages/adduser/ui.test.js]: should this and line 234, 236 be removed now?

File: app/pages/adduser/ui.test.js [hunk 2]
```diff
@@ -173,17 +173,69 @@ test('should navigate to confirmation page when form is filled out and addUser b
...
+    .expect(await extractInnerText(addUserPage.find('#emailAddress-error'))).contains('Email address already exists')
+    .expect(await extractInnerText(addUserPage.find('#emailAddress-error'))).contains('Email address format is not valid');
```
Comment (alibombaye) [inline app/pages/adduser/ui.test.js]: should also check that the input fields are of now of error and they have the errorMessages too

Comment (kat1906) [reply app/pages/adduser/ui.test.js]: this is checking the input fields

File: app/pages/adduser/ui.test.js (hunk 1 above)
Comment (kat1906) [reply app/pages/adduser/ui.test.js]: no? especially for the email address where 2 errors are put into 1 field

File: app/pages/adduser/ui.test.js (hunk 2 above)
Comment (kat1906) [reply app/pages/adduser/ui.test.js]: I couldn't extract the text from the summary when I tried either

Comment (kat1906) [reply app/pages/adduser/ui.test.js]: FYI we don't test this at all in MP

File: app/pages/adduser/ui.test.js (hunk 1 above)
Comment (alibombaye) [reply app/pages/adduser/ui.test.js]: ok in which case can we add that count back for firstName-error. I removed that yesterday

File: app/pages/adduser/ui.test.js (hunk 2 above)
Comment (alibombaye) [reply app/pages/adduser/ui.test.js]: It's ok I'll take a look at this

**Comment by kat1906** [comment]
😍

**Pull request merged**